Snakemake/rules/merge_samples_vcfs.smk

```diff
@@ -0,0 +1,99 @@
+#
+# Files must be at the vcf folder compressed and indexed
+#
+import os
+configfile: "../config/tsinfer.yaml"
+
+def list_full_paths(directory):
+    return [os.path.join(directory, file) for file in os.listdir(directory)]
+
+rule all:
+    input:
+        expand(contig['vcfDir'] + "Chr{chromosome}.vcf.gz", chromosome = range(1, config['noChromosomes'] + 1))
+
+workdir: config['workdir']
+vcfdir = config['vcfDir']
+
+VCFs =[x for x in list_full_paths(vcfdir) if x.endswith(".vcf.gz")]
+
+# creates a list of all files for that chromosome
+files_chr = [name in VCFs if '{chromosome}' in name]
+noFiles_chr = len(files_chr)
+identifier = range(noFiles_chr)
+
+if noFiles_chr == 1:
+    print('Nothing to merge')
+if noFiles_chr == 0:
+    print('Missing chromosome file')
+if noFiles_chr > 1:
+    print('Merging files...')
+
+    # Get sample list for each vcf
+    rule extract_vcf_samples:
+        input:
+            expand(file, file=files_chr)
```

Comment:
This I think will not work - expand works likes this: expand(file{chromosome}, chromosome = files_chr)